### ISSUE TITLE
"Wrong protocol version"

### ISSUE THREAD

**Issue by towerofnix** [FlyingPony/MineLittleProxy#1]
![](https://cloud.githubusercontent.com/assets/9948030/14069071/fb9c7344-f46a-11e5-81ea-2c8d0672f913.png)

> Wrong protocol version, expected: 47 and you are using: 107

Running Minecraft 1.9 on OS X El Capitan.

```
[Idle_1] Listening for clients
[Idle_2] Listening for clients
[Idle_3] Listening for clients
[Init] Proxy up
[Idle_1] Player LiamMorgaine connected to an idle server
```

**Comment by 109C** [comment]
This is probably because by default NMP starts up a 1.8 server. This should be an easy fix, I think.

Edit: Yes, it was because NMP defaults to 1.8.

**Comment by 109C** [comment]
Ok, this issue should be fixed by 668f23066802cbd3911547c53861b472d592cd82 .

**Comment by towerofnix** [comment]
Perfect!
